lib/elixir/lib/option_parser.ex

```diff
@@ -71,6 +71,9 @@ defmodule OptionParser do
   Note that you should only supply the `:switches` or the`:strict` option.
   If you supply both, an `ArgumentError` exception will be raised.
 
+    iex> OptionParser.parse(["--source", "lib"], switches: [source: :string], strict: [source: :string])
+    ** (ArgumentError) :switches and :strict cannot be given together
+
```

Comment:
Yes, I don't believe this example adds much, exactly because it is showing an error the user will get anyway if they "miss" this part of the documentation.